app/src/main/java/edu/cascadia/mobas/campusguidebook/EventsModal.java

```diff
@@ -0,0 +1,80 @@
+package edu.cascadia.mobas.campusguidebook;
+import androidx.room.Entity;
+import androidx.room.PrimaryKey;
+
+@Entity(tableName = "Events_Table")
+public class EventsModal {
```

Comment:
Recommend refactoring to `EventModel` or even just `Event` since each instance is just the data of a single event